.github/workflows/docker.yml

```diff
@@ -65,6 +65,9 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+permissions:
+  packages: write
+
```

Comment:
To be clear, unless this workflow triggers from a `push` event or `workflow_dispatch` event, etc rather than a `pull_request` event, `write` is not a permissions granted to the `github_token`.

Reply:
I think we can't have what we really need.
https://github.community/t/cant-authenticate-with-actions-token-for-pr-event/170752/31

https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

I think we can make it work for PRs made from `jupyter/docker-stacks` repo (not forks), but we can't have it for all the people. Or we will use some bad way with PAT tokens and basically any PR will be able to change our packages.